CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Pino is an OPEN Open Source Project
+
+## What?
+
+Individuals making significant and valuable contributions are given commit-access to the project to contribute as they see fit. This project is more like an open wiki than a standard guarded open source project.
+
+## Rules
+
+There are a few basic ground-rules for contributors:
+
+1. **No `--force` pushes** or modifying the Git history in any way.
```

Comment:
I think this rule is outdated - it's fine to modify git history on PR's (squashing) - just not on master